AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -363,46 +467,52 @@ private void openShareDialogue() {
         Log.d(LOG_TAG, "Share button clicked!");
         final Intent sendIntent = new Intent();
         sendIntent.setAction(Intent.ACTION_SEND);
-        sendIntent.putExtra(Intent.EXTRA_TEXT, callingContext.getGroupId());
+        sendIntent.putExtra(Intent.EXTRA_TEXT, callingContext.getJoinId());
```

Comment:
We should consider applying these renaming changes in main branch first to minimize changes later on when merging.

Reply:
done